modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -1190,12 +1190,12 @@ CompletableFuture<Ignite> joinClusterAsync() {
                                 metricManager,
                                 distributionZoneManager,
                                 computeComponent,
+                                volatileLogStorageFactoryCreator,
                                 replicaMgr,
                                 indexNodeFinishedRwTransactionsChecker,
                                 txManager,
                                 dataStorageMgr,
                                 schemaManager,
-                                volatileLogStorageFactoryCreator,
                                 outgoingSnapshotsManager,
```

Comment:
Should we also move `outgoingSnapshotManager` higher? It's about Raft, same as log storage factory creator

Reply:
`outgoingSnapshotManager` is only used in the `TableManager` which already starts after. `volatileLogStorageFactoryCreator` is used in `ReplicaManager`.